application/casp/publicKey_test.go

```diff
@@ -34,31 +32,28 @@ func TestGetTMPubKey(t *testing.T) {
 }
 
 func TestGetEthPubKey(t *testing.T) {
-	configuration.InitConfig()
-	configuration.SetConfig(test.GetCmdWithConfig())
+	test.SetTestConfig()
 
 	uncompressedPublicKeys, err := caspQueries.GetUncompressedEthPublicKeys()
 	require.Nil(t, err, "Failed to get casp Response")
-	ethPubliKey := uncompressedPublicKeys.PublicKeys[0]
-	ethKey := GetEthPubKey(ethPubliKey)
-	require.Equal(t, 20, len(crypto.PubkeyToAddress(ethKey)))
-	require.Equal(t, reflect.TypeOf(ecdsa.PublicKey{}), reflect.TypeOf(ethKey))
-	require.Equal(t, reflect.TypeOf(&big.Int{}), reflect.TypeOf(ethKey.X))
-	require.Equal(t, reflect.TypeOf(&big.Int{}), reflect.TypeOf(ethKey.Y))
-	require.NotNil(t, ethKey)
+	ethPublicKey := GetEthPubKey(uncompressedPublicKeys.Items[0])
+	require.Equal(t, 20, len(crypto.PubkeyToAddress(ethPublicKey)))
+	require.Equal(t, reflect.TypeOf(ecdsa.PublicKey{}), reflect.TypeOf(ethPublicKey))
+	require.Equal(t, reflect.TypeOf(&big.Int{}), reflect.TypeOf(ethPublicKey.X))
```

Comment:
I see this pattern in many places in the tests. Why do we this type check? We already have a strongly type golang type system.

Reply:
Can remove this.

Reply:
Please do.